region/norfair/east/Bat Cave.json

```diff
@@ -182,6 +182,31 @@
       "flashSuitChecked": true,
       "note": "Shoot diagonally to clear the Skree and shot block before jumping and sparking diagonally mid-air to the above area."
     },
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharging, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 2,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canLongChainTemporaryBlue",
+        "HiJump",
+        {"or": [
+          {"heatFrames": 570},
```

Comment:
Bouncing over the platforms can save a remorph, and half a second.
Stopping the shinecharge on the farthest right will get hit by a particle.  It's probably easy to avoid.